.ci/Jenkinsfile

```diff
@@ -89,24 +95,51 @@ pipeline {
       }
     }
     stage('Package') {
-      environment {
-        DOCKER_SECRET = 'secret/apm-team/ci/docker-registry/prod'
-        DOCKER_REGISTRY = 'docker.elastic.co'
+      when {
+        beforeAgent true
+        not { changeRequest() }
```

Comment:
Use https://www.jenkins.io/doc/book/pipeline/syntax/#built-in-conditions rather than the step `isPR()`